src/components/Filter.js

```diff
@@ -0,0 +1,50 @@
+import "./Filter.css";
+import { FaSearch } from "react-icons/fa";
+
+export default function Filter({ onStatusFilterChange, onNameFilterChange }) {
+  function handleFilterSelection(event) {
+    event.preventDefault();
```

Comment:
I don't think you need this here, right? Or does this event have any unwanted default things?